codemirror/widgets.py

```diff
@@ -159,13 +159,14 @@ def __init__(
             config.items(),
             [('mode', mode), ('theme', theme)])))
 
-    def render(self, name, value, attrs=None):
+    def render(self, name, value, attrs=None, renderer=None):
         u"""Render CodeMirrorTextarea"""
         if self.js_var_format is not None:
             js_var_bit = 'var %s = ' % (self.js_var_format % name)
         else:
             js_var_bit = ''
-        output = [super(CodeMirrorTextarea, self).render(name, value, attrs),
+        output = [super(CodeMirrorTextarea, self).render(
+                        name, value, attrs, renderer),
```

Comment:
I guess this code does not work with the old Django because of the number of arguments are changed.

Reply:
Would you like me to insert a version query? (As in, an `if` statement with both code paths)

Reply:
Yes please 👍

Reply:
Done.